src/stories/badge.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { ThemeProvider } from 'styled-components';
```

Comment:
Let's keep the stories next to the files they're for like we do for tests. It's just easier to keep track that way rather than having one flat directory with all the stories in it